dizoo/board_games/gomoku/envs/gomoku_env.py

```diff
@@ -0,0 +1,253 @@
+import gym
+import numpy as np
+import sys
+
+from dizoo.board_games.base_game_env import BaseGameEnv
+from ding.envs import BaseEnvTimestep
+from ding.utils import ENV_REGISTRY
+
+
+@ENV_REGISTRY.register('gomoku')
+class GomokuEnv(BaseGameEnv):
+    def __init__(self, cfg={}):
+        self.cfg = cfg
+        self.board_size = self.cfg.get('board_size', 15)
+        self.players = [1, 2]
+        self.board_markers = [
+            str(i + 1) for i in range(self.board_size)
+        ]
+        self.num_actions = self.board_size * self.board_size
+
+    @property
+    def current_player(self):
+        return self._current_player
+
+    @property
+    def current_opponent_player(self):
+        return self.players[0] if self.current_player == self.players[1] else self.players[1]
+
+    @property
+    def legal_actions(self):
+        legal_actions = []
+        for i in range(self.board_size):
+            for j in range(self.board_size):
+                if self.board[i][j] == 0:
+                    legal_actions.append(self.coord_to_action(i, j))
+        return legal_actions
+
+    def reset(self, start_player=0):
```

Comment:
as self.players = [1, 2], may the default value of start_player be 1 better than 0?